Improve formatting of `README.md`

Pranav-yadav opened the pull request:
The commit improves the formatting of `README.md` by adding `<kbd>`
elements for keyboard key instructions as well as the `<samp>`
elements for menu selections

Test: Inspected the rendered markdown page

Pranav-yadav commented:
> Thanks for your pull request! It looks like this may be your first contribution to a Google open source project. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> View this [failed invocation](https://github.com/google/orbit/pull/4017/checks?check_run_id=7526053724) of the CLA check for more information.
> 
> For the most up to date status, view the checks section at the bottom of the pull request.

Signed ! :)

florian-kuebler commented on README.md:
```diff
@@ -1,14 +1,24 @@
+The `div` reduces the vertical height.
+GitHub will autolink `img`, but won't produce a link when `href="#"`.
+-->
+<div><a href="#"><img src="contrib/docs/images/bumper.png" width="250"></a></div>
```
We should use the `======` or `------` markdown instruction instead of the bumper.
In particular, what are the licence for that bumper.png? Where is this coming from?

Pranav-yadav replied:
Yeah. The `bumper.png` is licensed (open-source) to Google, had taken it from carbon-lang repo.
Will delete it, let's keep things clutter free.

florian-kuebler reviewed (commented):
Thanks a lot for the change. Overall it looks good to me.
As said in the comment, I would rather not have an image for the bumper and just use markdown directly here.

Another minor comment: We do have some specific PR and commit policies, see [here](https://github.com/google/orbit/blob/main/CONTRIBUTING.md#pull-request-guidelines).
* In particular the PR should be a single commit.
* The description should contain a what/why.
* The commit message should have a proper commit header (max 72 chars, start with a capital letter, does not end with a period, and imperative style). E.g. "Improve formatting of README.md"
* The commit body should explain the what/why
* The commit body should contain a "Test:" stanza, how you tested your change. E.g. "Test: Inspected the rendered markdown page", maybe adding a screenshot.

Pranav-yadav commented:
<details>
<summary>Screenshots:</summary>
<img src="https://user-images.githubusercontent.com/55224033/181256054-170f2850-9228-4919-98c4-d5d5c5135ceb.png" >
<img src="https://user-images.githubusercontent.com/55224033/181256652-2b1dba32-4862-41e8-8c8a-6e5877ed4450.png" >
</details>

florian-kuebler reviewed (approved):
Looks good to me. Thanks for the change and for addressing the comments.

Pranav-yadav commented:
> Looks good to me. Thanks for the change and for addressing the comments.

Welcome.

Though I just had one doubt, why did you *initiated* all the actions (including builds) ?
I guess we should not *init* actions for doc changes explicitly if the doc change is done outside the main src.
I get the part of tests actions but, you know everyone should prefer *skipping ci* for changes like this one.

At least I prefer using `skip ci` inside commit messages for these type of changes. Why waste resources ? 🤗🌏
PS: You should consider configuring new label `doc-run` or something similar, for changes like this which will **NOT** *init* the build actions and could *init* actions specifically configured for doc changes like checking *semantics*, etc.
🤗🌏

Regards,
Pranav

florian-kuebler commented:
> > Looks good to me. Thanks for the change and for addressing the comments.
> 
> Welcome.
> 
> Though I just had one doubt, why did you _initiated_ all the actions (including builds) ? I guess we should not _init_ actions for doc changes explicitly if the doc change is done outside the main src. I get the part of tests actions but, you know everyone should prefer _skipping ci_ for changes like this one.
> 
> At least I prefer using `skip ci` inside commit messages for these type of changes. Why waste resources ? 🤗🌏 PS: You should consider configuring new label `doc-run` or something similar, for changes like this which will **NOT** _init_ the build actions and could _init_ actions specifically configured for doc changes like checking _semantics_, etc. 🤗🌏
> 
> Regards, Pranav

Yeah, it was definitely not necessary to run the CI. For external contributions, we actually never run the CI until a core developer explicitly triggers the CI.

We might want to have different settings of CIs (like a doc-run) set up in the future (e.g. to check for license files and formatting), but that is currently not high of a priority.
